### ISSUE TITLE
Resize?

### ISSUE THREAD

**Issue by zekenie** [elninotech/uppload#76]
Hi!

I'm wondering if you've thought about building in image resizing as a feature? I think I could hack it in the compress function with [pica](https://github.com/nodeca/pica) or do it on the backend.

**Comment by AnandChowdhary** [comment]
Yes, we have built-in resizing: https://uppload.js.org/compression

> To save on bandwidth and storage costs, you may want to upload resized and compressed images by default. If this is the case, you can configure Uppload to do so:

```ts
import { Uppload } from "uppload";

const uploader = new Uppload({
  maxSize: [800, 600]
});
```

> In the above example, images will be resized to a maximum of 800px in width and 600px in height. You can also specify only one of the two constraints using maxWidth or maxHeight.

**Comment by zekenie** [comment]
Ah, that makes sense! I think I figured `maxSize` would mean it would reject assets above that size, I didn't realize it would actually resize to fit. That's awesome
